src/main.cpp

```diff
@@ -977,6 +1158,17 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
                 __func__, hash.ToString(), FormatStateMessage(state));
         }
 
+        // Remove conflicting transactions from the mempool
+        BOOST_FOREACH(const CTxMemPool::txiter it, allConflicting)
+        {
+            LogPrint("mempool", "replacing tx %s with %s for %s BTC additional fees, %d delta bytes\n",
+                    it->GetTx().GetHash().ToString(),
+                    hash.ToString(),
+                    FormatMoney(nFees - nConflictingFees),
+                    (int)nSize - (int)nConflictingSize);
```

Comment:
(maybe out of scope for this PR)

Do we need a new signal here? How can the wallet detect removed (or replaced) transactions? Maybe the signal needs to be emitted from `void CTxMemPool::RemoveStaged(setEntries &stage)` (or nearby) to also include the mempool limiting behavior.